src/main/java/org/isf/sms/service/SmsOperations.java

```diff
@@ -34,7 +34,7 @@
 
 /**
  * @author Mwithi
- * @see org.isf.sms.model.Sms
+ * @see Sms
```

Comment:
Should it be like this?

```suggestion
 * @see {@link Sms}
```

Reply:
Done